Documentation/configuration/metrics.rst

```diff
@@ -50,6 +53,8 @@ Policy Imports
 --------------
 
 * ``policy_count``: Number of policies currently loaded
+* ``policy_regeneration_seconds``: Total time taken to regenerate policies
```

Comment:
Word this doc to make clear that you ignore regenerations that fail.
Or fix the implement to not ignore them.

Reply:
How about changing the wording as `Total time taken to successfully regenerate policies`

Reply:
done